app/controllers/ads_controller.rb

```diff
@@ -7,20 +7,24 @@ class AdsController < ApplicationController
   respond_to :html
 
   def index
+    @title = 'Ads'
```

Comment:
Can you change all the strings to use double quotes for consistency?

Reply:
Good point, done!